docs/includes/pairlists.md

```diff
@@ -77,17 +77,17 @@ Filtering instances (not the first position in the list) will not apply any cach
 ```
 
 !!! Note
-    `VolumePairList` does not support backtesting mode.
+`VolumePairList` does not support backtesting mode.
```

Comment:
Careful with these "Unintended" changes (probably done automatically by some IDE?).
This change (and all others that relate to lines after !!!) is wrong - the indent of 4 spaces is a must, otherwise rendering in the docs will break.

Reply:
noted, my bad 😅